server/services/deployment.spec.ts

```diff
@@ -51,4 +61,42 @@ describe('test deployment service', () => {
     expect(deployment._id).not.toBe(undefined)
     expect(deployment.uuid).toBe(testDeployment2.uuid)
   })
+
+  test('that we can email deployment owners about changes for multiple deployments', async () => {
+    const deployments: DeploymentDoc[] = [testDeployment, testDeployment2]
+    const spy: any = jest.spyOn(entityUtils, 'getUserListFromEntityList')
+    const userList: object[] = [
+      { name: 'Alice', email: 'alice@email.com' },
+      { name: 'Bob', email: 'bob@email.com' },
+    ]
+    spy.mockReturnValue(userList)
+
+    await emailDeploymentOwnersOnVersionDeletion(deployments, testVersion)
+
+    expect(emailClient.sendEmail).toHaveBeenCalledTimes(deployments.length * userList.length)
+  })
+
+  test('that we can email deployment owners about changes for a single deployment', async () => {
+    const deployments: DeploymentDoc[] = [testDeployment]
```

Comment:
Unfortunately our test data isn't fully representative of the full DeploymentDoc and won't have all the properties of a full document. It's probably best to leave it to infer the type itself at the moment, which will just be `any[]` as we haven't typed our test data.